packages/cli/src/imports.js

```diff
@@ -8,7 +9,23 @@ async function _import(modulePath) {
   try {
     m = await import(path.resolve(localNodeModules, modulePath))
   } catch (e) {
-    m = await import(modulePath)
+    try {
+      m = await import(modulePath)
```

Comment:
@pi0 I think the first `import` catch also need to exclude error other than `MODULE_NOT_FOUND`